### PR TITLE
feat: [Groot-1484] Update types in the CDA library with taxonomy assets

### PR DESCRIPTION
**Pull request by momenzadehvida**
Add [Taxonomy concept](https://www.contentful.com/help/taxonomies-overview/) links to assets.

### CODE REVIEW

File: test/integration/getEntries.test.ts [hunk 1]
```diff
@@ -334,15 +334,16 @@ describe('getEntries via client chain modifiers', () => {
         include: 5,
         'sys.id': entryWithResolvableLink,
       })
-      assertLocalizedEntriesResponse(response)
+      assertLocalizedEntriesResponse(response, true)
```
Comment (maxcheremisin) [inline test/integration/getEntries.test.ts:337]: this is now always called with `true`, should we remove the second arg?

**Review by marcolink — DISMISSED** [review]

File: test/integration/getEntries.test.ts (hunk 1 above)
Comment (momenzadehvida) [reply test/integration/getEntries.test.ts:337]: Nope, on `enforces entry.sys when query.select is defined` test is false.

Comment (maxcheremisin) [reply test/integration/getEntries.test.ts:337]: isn't it a different function, `assertCSMEntriesResponse`?

**Review by whitelisab — APPROVED** [review]

**Review by maxcheremisin — APPROVED** [review]
lgtm

**Review by whitelisab — APPROVED** [review]

**Pull request merged**